views/homepage.handlebars

```diff
@@ -118,6 +118,7 @@
             {{> battles}}
             {{/each}}
             {{/if}}
+    <iframe src="https://open.spotify.com/embed/playlist/63ZyQhDqXWsq0Z39oiyq8q" width="300" height="280" frameborder="0" allowtransparency="true" allow="encrypted-media"></iframe>
 {{!--       
```

Comment:
added a player to get some sound on our page.
this is just a widget. I think there is a way to play sound by button click.